src/Controller/PageAdminController.php

```diff
@@ -80,6 +81,7 @@ public function composeContainerShowAction(Request $request = null): Response
             throw new AccessDeniedException();
         }
 
+        /** @var Request $request */
```

Comment:
please remove null from argument because required by the code and so remove this new phpdoc